CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
```

Comment:
I know this was mentioned in the issue Neil logged. Do we have a good template & process for updating this? If note, I could see this being forgotten about and falling out of date pretty easily

Reply:
I agree, I have seen tools that auto generate the changelog from commit history, but we would need to change the way we structure commits to reflects more the: changes, chores, fixes etc..
Should ask @anmacdonald how he would like to do with it

Reply:
Not sure about toolkit, but we were pretty good at keeping DSA up to date. The rule was merges in v.next required a Changelog update.

https://github.com/Esri/dynamic-situational-awareness-qt/blob/main/RELEASE.md

Reply:
Applied Neil's suggestion! What is left to be done, is to populate it with this release changes